src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -226,7 +226,18 @@ public bool TryGet<T>(string name, out T obj)
             return TryGet(new StringSegment(name,0,name.Length),out obj);
         }
 
-        public bool TryGet<T>(StringSegment name, out T obj)
+	    public bool TryGetDetached(StringSegment name, out BlittableJsonReaderObject obj)
```

Comment:
This is using twice the memory, because it needs to re-read the data.
Why do we need that?